test/unit/ui/map.test.js

```diff
@@ -50,6 +50,19 @@ test('Map', (t) => {
         t.end();
     });
 
+    // t.test('map-specific token', (t) => {\
```

Comment:
How can we instantiate a map with a bad token?

Would like to add a test for the positive case, where it starts with `mapboxgl.accessToken = badtoken` but gets overwritten by a valid token inside the map options.

Reply:
> How can we instantiate a map with a bad token?
The unit tests don't exercise any access token specific logic. 

Map options are set-only with no ability to read back to options that were used to instantiate a map object. Since the access token is passed through to the `RequestManager` unit tests are probably suited to the behavior of that class.